### PR TITLE
docs(release-policy): split release policy

### PR DESCRIPTION
**Pull request by akeller**
Updating release policy page to point to new https://camunda.com/release-policy/

### CODE REVIEW

File: docs/reference/release-policy.md
Comment (christinaausley) [inline docs/reference/release-policy.md:33]: What do we mean by "follows above release policy" here? Do we mean follows **the** above release policy?

**Comment by christinaausley** [comment]
@akeller One minor adjustment in my latest commit, and one lingering comment. Will approve once resolved as there are no other adjustments on my end.

**Comment by akeller** [comment]
> and one lingering comment

@christinaausley I'm not seeing a comment, what am I missing?

**Comment by christinaausley** [comment]
> > and one lingering comment
> 
> @christinaausley I'm not seeing a comment, what am I missing?

Oops, comment was still in `Pending` as I added it to an official review.

**Comment by akeller** [comment]
Let's hold off on merging until @felix-mueller takes a look just to confirm the info is correct.

**Comment by akeller** [comment]
@felix-mueller can you review this before we merge? I've seen a lot of questions about release policy recently and I want to make sure this is correct before we push it.

**Review by felix-mueller — APPROVED** [review]

**Pull request merged**
